Resources/Locale/en-US/ghost/roles/ghost-role-component.ftl

```diff
@@ -122,6 +122,9 @@ ghost-role-information-skeleton-pirate-description = Cause chaos and loot the st
 ghost-role-information-skeleton-biker-name = Skeleton Biker
 ghost-role-information-skeleton-biker-description = Ride around on your sweet ride.
 
+ghost-role-information-closet-skeleton-name = Closet Skeleton
+ghost-role-information-closet-skeleton-description = Exist as a primordial force on the station. Live happily with the crew or wage sweet skeletal war. You have no allegiance.
```

Comment:
"wage sweet skeletal war" might be okaying antagonist activity so it should say you can be an antagonist if you choose

Reply:
yes these should be more explicit that its antag allowed imo